db/migrate/20190417081149_add_bookings_school_id_to_schools_school_profile.rb

```diff
@@ -0,0 +1,7 @@
+class AddBookingsSchoolIdToSchoolsSchoolProfile < ActiveRecord::Migration[5.2]
+  def change
+    add_column :schools_school_profiles, :bookings_school_id, :integer, null: false
+    add_foreign_key :schools_school_profiles, :bookings_schools
```

Comment:
is it worth adding an index on here for the bookings_school_id?

Reply:
Yep, done